docs/administration/service_management.md

```diff
@@ -1,132 +1,170 @@
+- **Filtering**: Services that eliminate certain files from being processed further.
+- **Internet Connected**: Services that need to connect to the internet to fetch updates or check data.
+- **Networking**: Services that analyze network traffic or related data.
+- **Static Analysis**: Services that analyze files without executing them.
+
+**Stage**
+
+The **Stage** column delineates at which point in the analysis pipeline a service will execute. The stages determine the sequence in which services operate, ensuring an organized and efficient workflow. Each service belongs to a single stage, executed in the following order:
+
+- **FILTER**: Initial filtering of files (e.g., Safelist). This stage determines whether subsequent stages will analyze a file.
+- **EXTRACT**: File extraction and unpacking (e.g., Extract, Unpacker). Services in this stage focus on decomposing files from archives or other compound formats.
+- **CORE**: Core analysis tasks. This stage is where the majority of services perform their analysis.
+- **SECONDARY**: Additional analysis performed with context from previous stages (e.g., VirusTotal). These services offer further insights based on the initial core analysis.
+- **POST**: Subsequent processing steps (e.g., URLCreator). Services in this stage act upon the results generated by earlier stages.
+- **REVIEW**: This final stage involves services that review the entire context of the analysis before performing their own tasks (e.g., Ancestry).
+
+**File Types**
+
+The **File Types** column specifies the Assemblyline file types that the service will accept for processing. This is defined using regular expressions. For instance:
+
+- `.*` will accept all files for analysis.
+- Specific regex patterns can be used to target certain Assemblyline file types, such as `executable/windows` for Windows executables.
+
+**External**
+
+The **External** column indicates whether the service sends data outside of Assemblyline's infrastructure. This is an important consideration for privacy and data security. An example is the VirusTotal service, which submits files to the VirusTotal platform for analysis.
+
+**Mode**
 
-The "Category" column refers to a service's category, which is a way of grouping services. Available categories out-of-the-box are Antivirus, Dynamic Analysis, External, Extraction, Filtering, Internet Connected, Networking, and Static Analysis. 
+The **Mode** column specifies how the service runs, providing information on whether it:
 
-The "Stage" column defines when a service should run. Possible stages are FILTER, EXTRACT, CORE, SECONDARY, POST and REVIEW. Stages are executed in the order defined in the list.
+- **Runs in Privileged Mode**: This means the service has elevated permissions, which may be necessary for certain types of analysis.
+- **Uses Service Server**: Indicates the service relies on a server-based backend for processing.
```

Comment:
Since I could not find proper definitions for both, can you provide rough ones that I can polish using an LLM?